merge/conflict_test.go

```diff
@@ -29,24 +29,26 @@ var (
 	_NS  = fieldpath.NewSet
 	_P   = fieldpath.MakePathOrDie
 	_KBF = fieldpath.KeyByFields
-	_SV  = value.StringValue
-	_IV  = value.IntValue
+	_V   = func(i interface{}) *value.Value {
```

Comment:
ah, the other definition copied this code. I still think it is weird to need a `*value.Value`; if you just needed a `value.Value` then this would just be `_V = value.NewValueInterface`